source/Halibut/Diagnostics/HalibutLimits.cs

```diff
@@ -31,6 +34,7 @@ static HalibutLimits()
         public static TimeSpan TcpClientHeartbeatReceiveTimeout = TimeSpan.FromSeconds(60);
         public static TimeSpan TcpClientConnectTimeout = TimeSpan.FromSeconds(60);
         public static TimeSpan PollingQueueWaitTimeout = TimeSpan.FromSeconds(30);
+        public static int LogStorageLimit = 5000;
```

Comment:
This number seems much too low. If I have 10,000 tentacles, every second one would have 1 message on average. Perhaps 100,000 is better.

Reply:
100,000 is close to where the original user had the issue (300,000 in their case). If customers have 10,000 tentacles (which is a rare case) then they can bump the `Halibut.LogStorageLimit` config value.

In any case the `GetLogs` call that means Halibut has to hold onto logs isn't even used in OD or Tentacle. If we could obsolete that call the better solution is just not store the logs.

Reply:
Unfortunately any Halibut settings need to be re-applied after a Server Upgrade :(